SYCL/BFloat16/bfloat16_example_gpu_aot.cpp

```diff
@@ -0,0 +1,10 @@
+///
+/// Check if bfloat16 example works using fallback libraries on GPU
+///
+
+// REQUIRES: ocloc, gpu
+// UNSUPPORTED: cuda
+// CUDA is not compatible with SPIR.
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend "-device *" %S/Inputs/bfloat16_example.cpp -o %t.out
```

Comment:
That's probably just me missing something.
The original test had much more RUN commands for various architectures. Does "device *" cover them all?

Reply:
The "*" device covers all GPU devices.